org.eclipse.m2e.core.ui/schema/m2e_component_search.exsd

```diff
@@ -0,0 +1,125 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Seems to me like an older version of the schema from org.eclipse.m2e.core.
Shouldn't it be removed?